src/helloworld.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react';
+
+class Hello extends React.Component {
+  render() {
+    return <h1>Hello {this.props.name}!</h1>;
+  }
+}
+
+
+export default Hello;
```

Comment:
The filename should always be equal to the classname of the component. So the file should be `Hello.js`.